### DISCUSSION TITLE
Route::get returns blank in Laravel 8 API

### DISCUSSION THREAD

**Discussion by albassam58** [laravel/framework#34495] [General]
- Laravel Version: 8.0
- PHP Version: 7.4.8
- Database Driver & Version: MySql 8

### Description:
GET method returns blank while the POST is okay.

### Steps To Reproduce:
1. Make an Route::apiResource('/test', UserController::class) route.
2. Add return response()->json(['test' => 'test']) in both index and store method of UserController.
3. Test your api routes using postman.
4. using the POST method with URL http://localhost:8000/api/test returns {"test":"test"}.
5. Using the GET method with URL http://localhost:8000/api/test returns blank.

**Comment by lagbox** [comment]
Works fine for me in 8.5.0

**Comment by albassam58** [comment]
I found out about the problem. I have a route {vue} in my web.php and it captured the /api/test. Thanks
